content/en/docs/security/hosting-best-practices.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Collector hosting best practices
+linkTitle: Collector hosting
+weight: 115
+---
+
+When setting up hosting for OpenTelemetry (OTel) Collector, consider these best
+practices to better secure your hosting instance.
+
+## Store data securely
+
+Your Collector configuration file might contain sensitive data, including
+authentication tokens or TLS certificates. See the best practices for
+[securing your configuration](/docs/security/config-best-practices/#create-secure-configurations).
+
+If you are storing telemetry for processing, make sure to restrict access to
+those directories to prevent tampering with raw data.
+
+## Keep your secrets secret
+
+Kubernetes [secrets](https://kubernetes.io/docs/concepts/configuration/secret/)
+are credentials that hold confidential data. They authenticate and authorize
+privileged access. If you're using a Kubernetes deployment for your Collector,
+make sure to follow these
+[recommended practices](https://kubernetes.io/docs/concepts/security/secrets-good-practices/)
+to improve security for your clusters.
+
+## Apply the rule of least privilege
+
+The Collector should not require privileged access, except where the data it's
+collecting is in a privileged location. For example, in order to get pod logs by
+mounting a node volume, the Collector daemonset needs enough privileges to get
+that data.
```

Comment:
This section is worded differently from the others, I would rephrase it so that it more explicitly tells you what to do (use the least amount of privilege required).

Reply:
Good catch on the change in wording! I used the example that @jpkrohling gave in a comment that I can't find now. I will work on rephrasing it to match the other examples on the page.

Reply:
I rewrote the section in 3b86e29. Let me know what you think. Thanks!